components/cdc/src/errors.rs

```diff
@@ -99,18 +99,27 @@ impl Error {
 
     pub fn into_error_event(self, region_id: u64) -> ErrorEvent {
         let mut err_event = ErrorEvent::default();
-        let mut err = self.extract_region_error();
-        if err.has_not_leader() {
-            let not_leader = err.take_not_leader();
-            err_event.set_not_leader(not_leader);
-        } else if err.has_epoch_not_match() {
-            let epoch_not_match = err.take_epoch_not_match();
-            err_event.set_epoch_not_match(epoch_not_match);
-        } else {
-            // TODO: Add more errors to the cdc protocol
-            let mut region_not_found = errorpb::RegionNotFound::default();
-            region_not_found.set_region_id(region_id);
-            err_event.set_region_not_found(region_not_found);
+        match self {
+            Error::Sink(SendError::Congested) => {
+                let mut congested = cdcpb::Congested::default();
+                congested.set_region_id(region_id);
+                err_event.set_congested(congested);
```

Comment:
How does old TiCDC handle this error? Will it see an empty error message?

Reply:
The old TiCDC treat this `congested` error as unknown error, print logs.